docs/reference/content/driver-async/getting-started/installation-guide.md

```diff
@@ -10,11 +10,16 @@ title = "Installation Guide"
 
 # Installation
 
+
 The recommended way to get started using one of the drivers in your project is with a dependency management system.
 
+{{% note class="important" %}}
+The MongoDB Async Driver requires Java 7 or greater.
+{{% /note %}}
+
 {{< distroPicker >}}
 
 ## MongoDB Async Driver
 The new asynchronous API that can leverage either Netty or Java 7's AsynchronousSocketChannel for fast and non-blocking IO.
 
-{{< install artifactId="mongodb-driver-async" version="3.0.0" >}}
+{{< install artifactId="mongodb-driver-async" version="3.0.0" dependencies="`bson`, `mongodb-driver-core`">}}
```

Comment:
How about we make the dependencies direct links to Sonatype also?  I don't think it will be obvious to people who don't get what's going on here how to get these.